addons/website_sale/static/src/js/website_sale_dashboard/website_sale_dashboard.js

```diff
@@ -59,6 +75,26 @@ export class WebsiteSaleDashboard extends Component {
 		) {
 			return 'text-danger';
 		}
-		return 'text-muted';
+		return '';
+	}
+
+	getDashboardCardAdditionalClass(filterName) {
+		const dashboardCardColor = {
+			'to_fulfill': 'purple',
+			'to_confirm': 'orange',
+			'to_invoice': 'cyan',
+		};
+		let dashboardCardClasses = [];
+		const noData = this.state.eCommerceData['overall'][filterName] == 0;
+		if(noData) {
+			dashboardCardClasses.push('bg-secondary text-secondary-emphasis disabled');
+		} else {
+			dashboardCardClasses.push('o_dashboard_card--' + dashboardCardColor[filterName]);
+		}
+		const filters = filterName == 'to_confirm' ? [filterName, 'from_website'] : [filterName, 'from_website','order_confirmed'];
+		if(this.isSameFilter(filters)) {
+			dashboardCardClasses.push('o_dashboard_card--active');
```

Comment:
If you apply my suggestion you should change this class as well.
```suggestion
			dashboardCardClasses.push('active');
```